IRelectra.h

```diff
@@ -33,26 +33,26 @@ class IRelectra
 public:
     // Initialize
     IRelectra(IRsend* remote);
-    
+
     // Send an IR packet with the given parameters.
     bool sendElectra(bool power, IRElectraMode mode, IRElectraFan fan, int temperature, bool swing, bool sleep);
 
 private:
     IRsend* _remote;
-    
-    // Encodes specific A/C configuration to a number that describes
+
+    // Encodes specific A/C configuration to a number that describes the command.
```

Comment:
`the configuration`.
Actually `Encodes specific A/C configuration to a number` is enough IMO.